tests/functional/aws-node-sdk/test/multipleBackend/objectCopy/azureObjectCopy.js

```diff
@@ -0,0 +1,245 @@
+const assert = require('assert');
+const async = require('async');
+
+const withV4 = require('../../support/withV4');
+const BucketUtility = require('../../../lib/utility/bucket-util');
+const constants = require('../../../../../../constants');
+const { config } = require('../../../../../../lib/Config');
+const { getAzureClient, getAzureContainerName, convertMD5 } =
+    require('../utils');
+const { createEncryptedBucketPromise } =
+    require('../../../lib/utility/createEncryptedBucket');
+
+const azureLocation = 'azuretest';
+const azureClient = getAzureClient();
+const azureContainerName = getAzureContainerName();
+
+const bucket = 'buckettestmultiplebackendobjectcopy';
+const key = `azureputkey-${Date.now()}`;
+const copyKey = `azurecopyKey-${Date.now()}`;
+const body = Buffer.from('I am a body', 'utf8');
```

Comment:
let's also test with a bigger body so we can make sure the streaming functionality is working correctly.  maybe 5 mb?